### PR TITLE
Install script

### PR DESCRIPTION
**Pull request by matthiaskrgr**

### CODE REVIEW

File: build_desura.sh [hunk 1]
```diff
@@ -1,43 +1,142 @@
...
+	echo "Making clean for cef..."
+	if [ -d "build_cef" ] ; then
+		cd ./build_cef/
+		make clean
+		cd ../
+	fi
+	if [ ! -d "build" ] ; then
+		echo "Removing install directory..."
+		rm -rf "install"
+	fi
```
Comment (karolherbst) [inline build_desura.sh]: - else: find install/ -type f -iname libcef_desura.so -delete ?

File: build_desura.sh [hunk 2]
```diff
@@ -1,43 +1,142 @@
...
+	echo "Making clean for desurium..."
+	if [ -d "build" ] ; then
+		cd ./build/
+		make clean
+		cd ../
+	fi
+	if [ ! -d "build_cef" ] ; then
+		echo "Removing install directory..."
+		rm -rf "install"
+	fi
```
Comment (karolherbst) [inline build_desura.sh]: same as above:
else
  find install/ -type f  ! -iname libcef_desura.so -delete
?

**Comment by karolherbst** [comment]
I think it would be better to make commands and subcommands.

```
./build_desura.sh compile desurium
./build_desura.sh compile cef rebuild desurium
```

**Comment by matthiaskrgr** [comment]
I'll have a look at this but it seems a bit more complicated and so I'd like to have this merged first :P

**Comment by matthiaskrgr** [comment]
> I think it would be better to make commands and subcommands.
> 
> ./build_desura.sh compile desurium
> ./build_desura.sh compile cef rebuild desurium

can't be done.

make fails to accept passed -j values such as "-j 2" (-j2 works) or  "-j2 check"

make: the `-j' option requires a positive integral argument

**Comment by matthiaskrgr** [comment]
unfinished progress in case someone finds a workaround: http://pastebin.com/qHH08MJn

**Pull request merged**